pkg/apis/eksctl.io/v1alpha5/validation.go

```diff
@@ -269,6 +269,17 @@ func validateNodeGroupBase(ng *NodeGroupBase, path string) error {
 		return fmt.Errorf("AMI Family %s is not supported - use one of: %s", ng.AMIFamily, strings.Join(supportedAMIFamilies(), ", "))
 	}
 
+	if ng.SSH != nil {
+		if enableSSM := ng.SSH.EnableSSM; enableSSM != nil {
+			if !*enableSSM {
+				return errors.New("SSM agent is now built into EKS AMIs and cannot be disabled")
```

Comment:
it can be disabled by using a custom AMI right?

Reply:
The SSM policy will always be added to the node role now, including for custom AMIs, just like we add certain policies like the CNI plugin policy (if `withOIDC` is not set), but it can be disabled/overridden by providing `attachPolicyARNs`.